HelloWorld.java

```diff
@@ -4,7 +4,8 @@ public static void main(String[] args) {
         // Prints "Hello, World" to the terminal window.
         System.out.println("Hello, World111");
         System.out.println("Hell");
-        System.out.println("Hellaaaaaa");
+        System.out.println("Hellaaaaaa");+
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:6730/coding_rules#rule_key=squid%3AS106)